src/page-test/TestPage.tsx

```diff
@@ -48,15 +48,15 @@ const TestPage: React.FC = () => {
       <h2>Test of news feed with data pulled from spreadsheet:</h2>
 
       {fetchedNewsFeed}
+      <h2>Test of impact dashboard</h2>
+      <div>
+        <ImpactDashboard />
+      </div>
```

Comment:
btw if you want, you can totally put this at the top of the test page! you don't have to put the news component I made first.